loadtest/control/noopcontroller/utils.go

```diff
@@ -0,0 +1,19 @@
+package noopcontroller
+
+import (
+	"math"
+	"math/rand"
+	"time"
+)
+
+func pickIdleTimeMs(minIdleTimeMs, avgIdleTimeMs int, rate float64) time.Duration {
+	// Randomly selecting a value in the interval
+	// [minIdleTimeMs, avgIdleTimeMs*2 - minIdleTimeMs).
+	// This will give us an expected value equal to avgIdleTimeMs.
+	// TODO: consider if it makes more sense to select this value using
+	// a truncated normal distribution.
+	idleMs := rand.Intn(avgIdleTimeMs*2-minIdleTimeMs*2) + minIdleTimeMs
+	idleTimeMs := time.Duration(math.Round(float64(idleMs) * rate))
+
+	return idleTimeMs * time.Millisecond
+}
```

Comment:
I think it would now make sense to export this function, move it in `loadtest/control/utils.go` and have both controllers use it. It's independent from the controller itself so we can easily avoid some repetition.